apps/mobile/src/components/widgets/accounts/sheets/add-account-sheet.layout.tsx

```diff
@@ -0,0 +1,78 @@
+import { RefObject } from 'react';
+import Animated, {
+  Extrapolation,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+} from 'react-native-reanimated';
+
+import { t } from '@lingui/macro';
+
+import {
+  Box,
+  CLOSED_ANIMATED_SHARED_VALUE,
+  PlusIcon,
+  Sheet,
+  SheetRef,
+  Text,
+  ThemeVariant,
+  WalletPlusIcon,
+} from '@leather.io/ui/native';
+
+import { AddWalletListItem } from '../../../add-wallet/add-wallet-list-item';
+
+const AnimatedBox = Animated.createAnimatedComponent(Box);
+
+interface AddAccountSheetBaseProps {
+  addAccountSheetRef: RefObject<SheetRef>;
+}
+
+interface AddAccountSheetLayoutProps extends AddAccountSheetBaseProps {
+  addToWallet(): unknown;
+  addToNewWallet(): unknown;
+  themeVariant: ThemeVariant;
+}
+export function AddAccountSheetLayout({
+  addAccountSheetRef,
+  addToWallet,
+  addToNewWallet,
+  themeVariant,
+}: AddAccountSheetLayoutProps) {
+  const animatedIndex = useSharedValue<number>(CLOSED_ANIMATED_SHARED_VALUE);
+
+  const animatedStyle = useAnimatedStyle(() => ({
+    marginTop: interpolate(animatedIndex.value, [-1, 0], [-200, 0], Extrapolation.CLAMP),
+    marginBottom: interpolate(animatedIndex.value, [-1, 0], [200, 0], Extrapolation.CLAMP),
+  }));
+
+  return (
+    <Sheet
+      isScrollView
+      animatedIndex={animatedIndex}
+      ref={addAccountSheetRef}
+      themeVariant={themeVariant}
+    >
+      <AnimatedBox style={animatedStyle}>
+        <Box p="5">
+          <Text pb="5" variant="heading05">
+            {t`Add account`}
+          </Text>
+          <Box flexDirection="column" gap="1">
+            <AddWalletListItem
```

Comment:
I copied this from the `add-wallet` sheet. I think I should refactor `AddWalletListItem` to share it. 

Maybe to a different component called `ListItem`? I was trying to think of a good name / place for it

Reply:
Sounds good, thanks! Naming can be hard sometimes. 

I'll refactor them in https://github.com/leather-io/issues/issues/138 

I can also update to migrate my `TouchableOpacity` code to `pressable`